tests/docker/config/construct-query.sparql

```diff
@@ -74,62 +139,53 @@ CONSTRUCT {
-                             ?circuitBrainRegionBN    :identifier  ?circuitBrainRegionId    ;
-                                                      :label       ?circuitBrainRegionLabel .
-  ?id  :derivation           ?derivation              .
-                             ?derivation              :identifier      ?entity     ;
-                                                      rdf:type         ?entityType ;
-                                                      :label           ?entityName .
-  ?id  :distribution         ?distribution            .
-                             ?distribution            :label           ?distributionName           ;
-                                                      :encodingFormat  ?distributionEncodingFormat ;
-                                                      :contentUrl      ?distributionContentUrl     ;
-                                                      :contentSize     ?distributionContentSize    .
-  ?id  :image                ?image                   .
-                             ?image                   :identifier      ?image                  ;
-                                                      :about           ?imageAbout             ;
-                                                      :repetition      ?imageRepetition        ;
-                                                      :stimulusType    ?imageStimulusTypeLabel .
-  ?id  :series               ?series                  .
-                             ?series                  :value           ?seriesValue      ;
-                                                      :unit            ?seriesUnit       ;
-                                                      :statistic       ?seriesStatistic  .
+  ## Neuron density
+  ?id  :neuronDensity      ?neuronDensityBN       .
+                           ?neuronDensityBN       :value       ?neuronDensityValue   ;
+                                                  :unit        ?neuronDensityUnit    ;
+                                                  :nValue      ?neuronDensityNValue  ;
+                                                  :label       ?neuronDensityLabel   .
+
+  ## Simulation campaigns
+  ?id  :config               ?campaignConfigBN        .
+                             ?campaignConfigBN        :identifier      ?campaignConfigId   ;
+                                                      :label           ?campaignConfigName .
+
+  ## Simulation campaigns / simulations
+  ?id  :attrs                ?attrs                   .
+                             ?attrs                   ?attrs_prop       ?attrs_value .
+
+  ?id  :coords               ?coords                   .
+                             ?coords                   ?coords_prop       ?coords_value .
+
+  ## Simulations
+  ?id  :campaign             ?campaignBN               .
+                             ?campaignBN               :identifier      ?campaignId   ;
```

Comment:
Is it possible to get rid of the blank node here? Just using `?campaignId` instead. Eg. what is done for species and others